.github/workflows/tests_sources.yml

```diff
@@ -45,14 +45,41 @@ jobs:
         export OPENDR_HOME=$PWD
         pip install -r tests/requirements.txt
         python -m unittest discover -s tests
-    - name: Test API
-      if: ${{ matrix.os == 'ubuntu-20.04' && contains(github.event.pull_request.labels.*.name, 'test tools') }}
+  test-tools:
+    needs: cleanup-runs
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'test tools') }}
+    strategy:
+      matrix:
+        package:
+          - engine
+          - utils
+          - perception/activity_recognition
+          - perception/compressive_learning
+          - perception/face_recognition
+          - perception/heart_anomaly_detection
+          - perception/multimodal_human_centric
+          - perception/object_tracking_2d
+          - perception/object_detection_3d
+          - perception/pose_estimation
+          - perception/speech_recognition
+          # - perception/object_tracking_3d
+    runs-on: ubuntu-20.04
```

Comment:
Note that at some point we will run the tests on others OS too.
But this can be changed later.

Reply:
Yes, it is trivial to extend the matrix runner in the samme manner as it was done for the 'test-sources' workflow

Reply:
I have update the code to use a os matrix (with ubuntu 20.04 only) in commit 89b95cb2